tendrl/ceph_integration/objects/pool/atoms/check_pool_available/__init__.py

```diff
@@ -0,0 +1,43 @@
+import gevent
+
+from tendrl.ceph_integration.objects.pool import Pool
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.commons.objects import AtomExecutionFailedError
+
+
+class CheckPoolAvailable(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(CheckPoolAvailable, self).__init__(*args, **kwargs)
+
+    def run(self):
+        retry_count = 0
+        while True:
+            pools = NS._int.client.read(
```

Comment:
ok that is because pool id is not available as response of pool create rados request?